docs/admin/install.rst

```diff
@@ -389,9 +391,77 @@ This command will take a considerable amount of time and approximately 4GB of ba
+
+Once the update check is complete, the SecureDrop Client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+
+Post-Install Configuration
+--------------------------
+Once you have verified that you can log into the SecureDrop Client, you may want to consider making additional configuration changes:
+
+- setting up password management
+- enabling selected clipboard usage, if appropriate
+- enabling copying of system logs to a networked VM.
+
+By default, copy and paste is fully disabled for all SecureDrop Workstation VMs. Copying files from or to SecureDrop VMs using ``qvm-copy`` is also disabled.
+
+.. _Password Management Section:
+
+Password management
+~~~~~~~~~~~~~~~~~~~
+If you have installed Qubes with the default configuration, you should have a VM called ``vault``, which is a networkless VM based on the ``fedora-31`` template. You can use this VM to store SecureDrop login credentials or the passphrases for encrypted export USB drives.
+
+Note that all users of this workstation will have access to the password database. If the workstation will be used by more than one journalist, you may want to encourage the use of a smartphone password manager for managing individual login credentials.
+
+The ``fedora-31`` template includes the KeePassXC password manager. It is not listed in the application menu for ``vault`` by default; you can add it by selecting it from the list of available apps in **Q > Domain: vault > Qube Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
+
+After confirming this change, you can launch KeePassXC from the **Domain: vault** menu. On first start, it will suggest enabling automatic updates, which you should decline: ``vault`` is networkless, so the built-in update check will fail; the app will be updated through system updates instead.
+
+You can now choose between creating a new password database or importing an existing one.
+
+..  important::
+
+    If you import an existing database, make sure you import only the passwords you need. The password database stored on the *Admin Workstation* USB is intended to hold credentials required for the *administration* of the SecureDrop servers, which journalist users should not have access to. In contrast, a *Journalist Workstation* USB is intended to only hold the SecureDrop login credentials for a given journalist user of the system.
+
+    We recommend reviewing each section of the password database before giving other users access to the workstation.
```

Comment:
This is more an overall style point - the use of "we" was avoided in the rest of this doc, it might be worthwhile being consistent with that. (Who is this mysterious "we" anyway? They never introduce themselves, that's a little creepy.)

Reply:
For comparison, here are all instances of the phrase "We recommend" in the SecureDrop core docs. In my view, use of that phrase is reasonable -- even if the user does not know exactly who "We" is, it implicitly speaks to recommendations from the developers of the system. 

There are alternative phrases, but the use of passive voice ("It is recommended") that's often substituted as an alternative has its own problems; it's impersonal and begs the question "by whom".

<details>
<summary>Click to expand</summary>

```
ossec_alerts.rst
359:SMTP relay securely. We recommend doing this before running the

set_up_tails.rst
28:We recommend creating an initial Tails Live DVD or USB, and then using
95:   passphrase that's easy to write down or remember. We recommend using
105:   passphrases. We recommend using `spaced-repetition

journalist.rst
232:    in a given session. We recommend storing this passphrase in your own
445:and flac. We recommend using this and other tools to work with documents within

servers.rst
36:them to your Admin Workstation. We recommend saving them to the
257:several times during SecureDrop's installation. We recommend that the

passphrase_best_practices.rst
25:   .. tip:: We recommend reviewing your passphrase (e.g. by ensuring that you
79:user role in :doc:`passphrases`. We recommended using the KeePassXC password

https_source_interface.rst
82:   unauthorized access. We recommend doing this entire process on the Admin

hardware.rst
206:We recommend using one or multiple designated USB drives as the *Transfer
328:indistinguishable from each other unless you label them. We recommend buying
383:NUC before it can be used. We recommend:
562:persistent partition. We recommend getting something in the 16-64GB
581:We recommend the `pfSense SG-3100

admin.rst
31:manual action if requested in the `SecureDrop Release Blog`_. We recommend
357:server, their encrypted files may still exist in backups. We recommend
394:instance via the Admin Interface. We recommend a size of ``500px x 450px``.
453:We recommend admins establish a secure communication pipeline to alert fellow admins

update_bios.rst
59:Power off the *Monitor Server*. We recommend plugging it into an uninterrupted power supply (UPS). Plug in the keyboard, monitor, and USB key, and power on the server, then press F7 when prompted to enter the BIOS Update tool.

overview.rst
150:admin. We recommend that you set aside at least a week to

includes/encrypting-drives.txt
4:   encrypted and protected with a secure passphrase. We recommend using the

includes/otp-app.txt
1:.. tip:: We recommend using FreeOTP (available `for Android <https://play.google.com/store/apps/details?id=org.fedorahosted.freeotp>`__

set_up_transfer_and_export_device.rst
181:We recommend using a fully encrypted USB drive for copying files off the
190:We recommend the use of VeraCrypt. It is actively maintained cross-platform
206:   We recommend against installing the VeraCrypt software on the
222:that the drive must work across platforms (including Tails). We recommend

network_firewall.rst
279:#. Set a strong admin passphrase. We recommend generating a strong passphrase
380:	  We recommend keeping it simple and using the same DNS

source.rst
91:We recommend conducting all research related to your submission in Tor Browser.
167:   you adjust the Security Setting again. We recommend keeping the setting at

rebuild_admin.rst
432:We recommend completing the following tasks after the rebuild:

glossary.rst
119:network device. We recommend physically removing the drive and any
147:authentication codes. We recommend using one of:

update_tails_usbs.rst
29:We recommend that you :ref:`back up your existing configuration <backup_workstations>` and update as soon as possible. It usually takes some time for updates to download, so keep that in mind when choosing when to update.

development/setup_development.rst
32:We recommend using the stable version of Docker CE (Community Edition) which can
180:We recommend using the latest stable version of Vagrant, ``1.8.5`` at the time
208:provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
263:The version of rsync installed by default on macOS is extremely out-of-date, as is Apple's custom. We recommend using Homebrew_ to install a modern version (3.1.0 or greater): ``brew install rsync``.
```
</details>

Reply:
Discussed that a bit more, I agree it makes sense to get rid of some of the "we recommend"s that are specifically in the context of step-by-step instructions where it can be distracting and suggests more choice than is warranted.

Reply:
Done in dd9242951e399f860eae68bf2ed42b75cf0d11ad and f28f25ed3f46229b5b55b842c46b644065912622